SnapshotTests/SnapshotTestCase.swift

```diff
@@ -50,15 +50,32 @@ extension SnapshotTestCase {
         }
     }
 
-    func nameForDevice(baseName: String? = nil) -> String {
+    func nameForDevice(_ baseName: NameForDevice = .portrait) -> String {
```

Comment:
I think this method should have documentation.

Reply:
fixed [866122b](https://github.com/salemove/ios-sdk-widgets/pull/778/commits/866122b5b8943a8a5435a943d4ffd351736af208)